src/ssmDocument/commands/createDocumentFromTemplate.ts

```diff
@@ -0,0 +1,119 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as vscode from 'vscode'
+import * as YAML from 'yaml'
+import { getLogger, Logger } from '../../shared/logger'
+import * as picker from '../../shared/ui/picker'
+import { openAndSaveDocument } from '../util/util'
+
+import { getDocumentTemplate } from 'aws-ssm-document-language-service'
+
+export interface SsmDocumentTemplateQuickPickItem {
```

Comment:
I don't know if you're getting any lint warnings, but this should probably extend vscode.QuickPickItem, because the underlying picker expects that type

Reply:
I will add this.